runner/src/main/scala/stress/chain/Tasks.scala

```diff
@@ -66,6 +66,42 @@ object Tasks {
       50d -> setTeamOnRandomTask(None)
     )
 
+  def commentOnTasks(): ChainBuilder =
+    exec(session => {
+      session.set("commentOnTasksBody",
+        s"""{"taskIds":[${session.taskIds.map(taskId => s""""${TaskIds.toDomainId(taskId)}"""").mkString(",")}],
+           |"commentText":"This task needs some comments"}""".stripMargin)
+    })
+    .exec(
+      http("Comment on tasks")
+        .post("/tasks/comments")
+        .body(StringBody("${commentOnTasksBody}"))
+        .asJSON
+    )
+
+  def changeAssignmentOnTasks(): ChainBuilder = exec(session => {
+    session.set("changeAssignmentOnTasksBody",
+        s"""{"taskIds":[${session.taskIds.map(taskId => s""""${TaskIds.toDomainId(taskId)}"""").mkString(",")}],
+           |"team":"Release Admin", "owner": "admin"}""".stripMargin)
+    })
+    .exec(
+      http("Change assignment on tasks")
+        .put("/tasks/reassign")
+        .body(StringBody("${changeAssignmentOnTasksBody}"))
+        .asJSON
+    )
+
+  def removeTasks(): ChainBuilder = exec(session => {
```

Comment:
If I'm not mistaken, most scenarios don't actually delete stuff from a release, but query stuff or edit, so if you invoke this are you sure that after a while you won't delete all available tasks and that this scenario will be useless?